### DISCUSSION TITLE
API for mining summary?

### DISCUSSION THREAD

**Discussion by secondl1ght** [janoside/btc-rpc-explorer#374] [Q&A] [answered]
Is there an API for the mining summary section? Where does it pull that data from? I have an idea for a website that I want to build that would rely on using an API to see which mining pool mined each block in real time.

**Comment by janoside** [comment]
@secondl1ght No, currently there's no API endpoint for this data. But I created a new Issue #383 to create one. Coming soon!

**Comment by janoside** [comment, marked as answer]
Note: this is done and can be seen at the below URL:

https://bitcoinexplorer.org/api/mining/miner-summary?since=1d
